extension/src/popup/components/accountMigration/ReviewMigration/index.tsx

```diff
@@ -0,0 +1,248 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Badge, Button, Checkbox, Loader } from "@stellar/design-system";
+import { getAccountInfo, getMigratableAccounts } from "@shared/api/internal";
+import { useTranslation } from "react-i18next";
+import { BigNumber } from "bignumber.js";
+import { Field, FieldProps, Form, Formik } from "formik";
+import { object as YupObject, boolean as YupBoolean } from "yup";
+
+import { ROUTES } from "popup/constants/routes";
+import { BASE_RESERVE } from "popup/constants/transaction";
+
+import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+import { saveIsMergeSelected } from "popup/ducks/transactionSubmission";
+
+import { truncatedPublicKey } from "helpers/stellar";
+import { navigateTo } from "popup/helpers/navigate";
+import { IdenticonImg } from "popup/components/identicons/IdenticonImg";
+
+import { MigrationHeader, MigrationParagraph } from "../basics";
+
+import "./styles.scss";
+
+type AccountList = {
+  publicKey: string;
+  name: string;
+  trustlines: number;
+  dataEntries: number;
+  xlmBalance: string;
+  isSigner: boolean;
+  minBalance: string;
+}[];
+
+interface FormValues {
+  isMergeSelected: boolean;
+}
+
+const AccountInfo = ({
+  account,
+}: {
+  account: { publicKey: string; name: string };
+}) => (
+  <div className="ReviewMigration__account">
+    <div className="ReviewMigration__account__identicon-wrapper">
+      <IdenticonImg publicKey={account.publicKey} />
+    </div>
+    <div className="ReviewMigration__account__name">{account.name}</div>
+    <div className="ReviewMigration__account__public-key">
+      ({truncatedPublicKey(account.publicKey)})
+    </div>
+  </div>
+);
```

Comment:
These subcomponents will likely be broken out into their own files as they'll be needed again later in the flow